docs/content/en/docs/getting-started/_index.md

```diff
@@ -483,4 +483,4 @@ See [KeptnApp Reference page](../yaml-crd-ref/app.md) for more information.
 
 Keptn can run pre and post deployment tasks and SLO evaluations automatically.
 
-Continue the Keptn learning journey by [adding deployment tasks](../tutorials/add-tasks.md).
+Continue the Keptn learning journey by [adding deployment tasks](../tutorials/tasks/_index.md).
```

Comment:
https://github.com/keptn/lifecycle-toolkit/pull/2088 deletes the "Tutorials" section.  The exercise that is being referenced is new to me and I can't find the PR where it was added -- aargh!  I think we need to make "Getting started" a section that has the metrics/observability exercise that is currently called just "Getting started" with a title like "Implement Metrics and Observability" and this one with the title it has.

Reply:
Talked to Anna and we're going to merge this PR then go back and clean up the structure of the docs in other PRs.